network/nxos/nxos_vrrp.py

```diff
@@ -57,6 +57,12 @@
             - clear text authentication string
         required: false
         default: null
+    admin_state:
+        description:
+            - Used to enable or disable the VRRP process
+        required: false
```

Comment:
Can you please add
`version_added: "2.2"`
at the same level as `description` and `required`